core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/ConcurrentKeyExtentCache.java

```diff
@@ -83,8 +83,8 @@ protected void updateCache(KeyExtent e) {
 
   @VisibleForTesting
   protected Stream<KeyExtent> lookupExtents(Text row) {
-    return TabletsMetadata.builder().forTable(tableId).overlapping(row, null).checkConsistency()
-        .fetch(PREV_ROW).build(ctx).stream().limit(100).map(TabletMetadata::getExtent);
+    return TabletsMetadata.builder(ctx).forTable(tableId).overlapping(row, null).checkConsistency()
```

Comment:
Would it improve readability if one of context or ctx was used consistently? The first two files reviewed used context, some of the rest seem to be using ctx - don't have a strong preference for either - context takes less mental translation, but ctx is almost as clear, and shorter.

Reply:
Possibly but I think that is unrelated to this change.